bazel/repositories.bzl

```diff
@@ -867,6 +867,11 @@ def _com_github_google_jwt_verify():
         actual = "@com_github_google_jwt_verify//:jwt_verify_lib",
     )
 
+    native.bind(
+        name = "simple_lru_cache_lib",
```

Comment:
Do we want LRU cache for other things in Envoy eventually as well, e.g. async client cache? CC @chaoqin-li1123 

If so, I'd suggest we move this code into Envoy proper. I'm surprised this isn't already available in some existing dep.

Reply:
Hi @htuch 
should we use a separate PR or issue to move LRU cache code into Envoy?

Reply:
We already have LRU in envoy( https://github.com/envoyproxy/envoy/blob/1d1b708c7bf6efa02c41d9ce22cbf1e4a1aeec2c/source/common/stats/recent_lookups.cc), the key of the cache is limited to string though.

Reply:
I just checked recent_lookups.cc code.  It just keeps track of recent used keys.  It is NOT a cache map which you can store {key, value} pairs with O(1) lookup speed and with LRU auto purge.

Reply:
Yes, separate PR please. Other places I can think this would be useful is when we do RLS quota caching. I know dynamic forward has a cache, but it doesn't look like it does LRU today (maybe it would if we had this support?). Anyway, I can see an LRU map being something generally useful in the code base.

Reply:
https://github.com/envoyproxy/envoy/issues/16108  to track this work.  You can assign it to me

Reply:
This is my only concern w.r.t deps, any update?

Reply:
Hmm,  the issue has been closed.  I did not have time to work on that.

@mk46 do you have time to work on [#16108](https://github.com/envoyproxy/envoy/issues/16108)?  The work is to move lru_cache code into envoy,  maybe /source/common/lru_cache and add some unit-tests?

Reply:
@qiwzhang Yes, I will work on that. Feel free to assign me. Thanks!